example/example.dart

```diff
@@ -52,3 +54,32 @@ String scopedLibrary() {
   final library = Library((b) => b.body.addAll(methods));
   return _dartfmt.format('${library.accept(DartEmitter.scoped())}');
 }
+
+/// Outputs:
+///
+/// ```dart
+/// enum Unit {
+///  @JsonKey('m')
+///  Metric,
+///  @JsonKey('i')
+///  Imperial
+/// }
+/// ```
+String jsonEnum() {
+  final values = <EnumValue>[
+    EnumValue((b) => b
+      ..name = 'Metric'
```

Comment:
This should be `'metric'` in our examples.

Reply:
What do you mean? Should I rename the enum values to be lowercase?

Is that what should it look like?
```dart
enum Unit {
 @JsonKey('m')
 metric,
 @JsonKey('i')
 imperial
}
```

Reply:
Yes. We always follow best practices from Effective Dart in our examples.

https://dart.dev/guides/language/effective-dart/style#do-name-other-identifiers-using-lowercamelcase

Reply:
Oh, that is right, it should be fixed now.